instructions/docs/getting-started.md

```diff
@@ -0,0 +1,49 @@
+# Getting Started
+
+## Background
+
+In this section, you will prepare your AWS Environment to work with the Busy Engineer's Document Bucket. You will:
+
+1. Deploy a CloudFormation stack to launch a Cloud9 IDE
+1. Launch the Cloud9 IDE that you will use to work with the rest of the workshop
+1. Launch resource stacks using the AWS Cloud Development Kit (CDK), including
+    * The `BusyEngineersDocumentBucket` stack, with your DynamoDB table and S3 bucket
+    * One CMK in one region, called `Faythe`, that you will use in encryption and decryption operations
+    * One CMK in another region, called `Walter`, that you will use in encryption and decryption operations
+1. Bootstrap the development environment in Cloud9
+1. Start the workshop!
+
+## Make the Change
+
+1. Sign in to your AWS Account for the workshop
+    * Make sure this is **not** a production account!
+1. [Click this link to load the CloudFormation template for your Cloud9 IDE](https://us-east-2.console.aws.amazon.com/cloudformation/home?region=us-east-2#/stacks/quickcreate?templateUrl=https%3A%2F%2Fbusy-engineers-cfn.s3.us-east-2.amazonaws.com%2Fdocument-bucket-cloud9-bootstrap.yaml&stackName=BusyEngineersDocumentBucketEnvironment)
+1. Click **Create Stack** to launch the stack.
+1. [Open the Cloud9 Console](https://us-east-2.console.aws.amazon.com/cloud9/home?region=us-east-2#) to find your Cloud9 IDE.
+1. Click **Open IDE** to launch Cloud9
+    * The repository will check out to `~/environment/workshop` automatically
+1. `cd ~/environment/workshop`
+1. Execute `make bootstrap` to set up your workshop environment
+
+## Try it Out
+
+* `make bootstrap` will take a while. Thank you for your patience while it sets up the environment for you.
```

Comment:
can you be more specific about how much time it took for you and what it's setting up?

Reply:
Ah shoot missed this. Will fix